modules/ROOT/pages/_partials/usage-prerequisites.adoc

```diff
@@ -0,0 +1,3 @@
+// tag::permissions[]
+To view usage reports, you must have the Usage Viewer permission over your root organization.
```

Comment:
Is "over" the right word here (and elsewhere)? I usually see it as "for".

Reply:
Agree. Updated.